data/management/commands/fetch_deployed_data.py

```diff
@@ -34,9 +36,18 @@ def handle(self, *args, **options):
             except Exception:
                 if upstream_deploy_url:
                     r = requests.get(upstream_deploy_url + '/' + filename)
-                    r.raise_for_status()
+                    try:
+                        r.raise_for_status()
+                    except Exception:
```

Comment:
you are not using the exception, soo ..

`if not allow_failure: r.raise_for_status()`

??

But I need to see this justification for allowing the failure....